elements/pfe-navigation/src/pfe-navigation.js

```diff
@@ -672,6 +670,7 @@ class PfeNavigation extends PFElement {
     }
 
     // Copy light DOM menu into new wrapper, to be put in shadow DOM after manipulations
+    // @todo: look into replacing the light dom nav menu with the shadow dom copy instead of copying the light dom into the shadow dom and having two navs at the same time with the same content, not great for a11y to have two copies of the same menu, the two copies are causing tab order issues on firefox
```

Comment:
We could add a style to `display: none` all ancestors of `pfe-navigation` in the light DOM after the component is mounted?

Reply:
@wesruv that might work, seems worth a try. Any ideas on what is the best way to adds some styles to a web component only if it mounts successfully? Not sure what the logic is for determining when a web component mounts. I think we only need to hide the nav in the light dom and not hide the slots.

Reply:
I've been styling `.PFElement`, which is a class that gets added to mounted pfe components, but I never asked @castastrophe (or anyone else) maybe there's something better?

Reply:
In the template, you can do: `<slot hidden></slot>` and add styles for `[hidden]` to your sheet to display none.  This will essentially hide all the light DOM.

Reply:
thanks @castastrophe! hidden attr sounds good to me. really want to get used to using that attr when I need to display: none content. How does one go about adding an attribute to the light dom once the component mounts, what is the logic to know when has actually successfully mounted? Do you have any examples of that I could take a look at?